todo-list-ui/src/app/confirmation-modal/confirmation-modal.component.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Component,
+  EventEmitter,
+  inject,
+  Input,
+  Output,
+  TemplateRef,
+  ViewChild,
+} from '@angular/core';
+import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'app-confirmation-modal',
+  standalone: true,
+  templateUrl: './confirmation-modal.component.html',
+})
+export class ConfirmationModalComponent {
+  @Input({ required: true }) message?: string;
```

Comment:
This is fine but in the future you may want to use content-projection https://angular.dev/guide/components/content-projection instead of Inputs for these use cases